src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotCorrect = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (robotCorrect[value]) {
```

Comment:
Instead of checking if `robotCorrect[value]` exists, use `robotCorrect.hasOwnProperty(value)` to avoid issues with inherited properties.